core/pom.xml

```diff
@@ -34,7 +34,11 @@
 
     <dependencies>
         <dependency>
-            <groupId>org.apache.james</groupId>
+            <groupId>${project.groupId}</groupId>
```

Comment:
I'm really not fan of thisdependency....

It means I can't use core models in mailbox-api. Isn't it a bit sad?

I would prefer to reverse that dependency: Have an interface holding the delimitier in James core... We could then re-use it in MailboxConstants.

Reply:
For me it's the name of the `james-core` project which is wrong.
It's more `james-base-pojos`, which is the truth (only `MailAddress` & `User` class in this project).
Then, it's logic that the API depends on this project.

Reply:
Your POJO project depends on an API project.

What if later on I want to use the pojos in my api? Or in my API package for my basic API logic?

Reply:
The `User` class needs information from the `MailboxConstants` class, which is in _apache-james-mailbox-api_ ...
I will try to put all the common classes in the _james-core_ project.

Reply:
Maybe that constant should not be in MailboxConstants class, no?

I mean it starts having widder impact that "Just being a MailboxConstants".

Maybe we can introduce a "JamesConstants" class?